experimenter/experimenter/nimbus_ui_new/forms.py

```diff
@@ -35,6 +36,58 @@ def save(self, *args, **kwargs):
         return experiment
 
 
+class NimbusSerializerErrorMixin:
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.validation_errors = {}
+
+    def clean(self):
+        cleaned_data = super().clean()
+        if self.show_validation_errors:
```

Comment:
See here how we rely on `self.show_validation_errors`, but it's not set/defined on this mixin, but on the other one?  That suggests to me that these two mixins must always be used together, in which case they should probably just be a single mixin.